packages/jest-haste-map/src/index.js

```diff
@@ -429,21 +437,28 @@ class HasteMap extends EventEmitter {
 
     const fileMetadata = hasteMap.files[filePath];
     const moduleMetadata = hasteMap.map[fileMetadata[H.ID]];
+    const sha1 = this._options.sha1 && !fileMetadata[H.SHA1];
```

Comment:
This is a boolean, but the variable is called "sha1". This may be more readable if it was called `shouldComputeSha1` or similar.